src/Compilers/CSharp/Portable/Parser/Lexer_RawStringLiteral.cs

```diff
@@ -120,7 +120,7 @@ private void ScanRawStringLiteral(ref TokenInfo info, bool inDirective)
                 };
             }
 
-            info.Text = TextWindow.GetText(intern: true);
+            info.Text = this.GetInternedLexemeText();
```

Comment:
References to `TextWindow.GetText(intern: true)` and `TextWindow.GetText(intern: false)` were mechanically updated to `this.GetInternedLexemeText()` and `this.GetNonInternedLexemeText()` mechanically.